userdetails-plugin/grails-app/views/registration/createAccount.gsp

```diff
@@ -422,6 +446,110 @@
              document.getElementById("mfa").hidden = true
          });
 
+         $("#requestCode").click(function(e) {
+             var newEmail = $("#newEmail").val();
+             if(newEmail == null || newEmail === "") {
+                 document.getElementById("newEmailMessage").innerHTML = "Invalid email"
+                 document.getElementById("newEmailMessage").style.color = "red"
+                 document.getElementById("newEmailMessage").hidden = false
+             }
+             else {
+                 $.ajax({
+                 url: "${createLink(uri:'/registration/update')}?email=" + newEmail,
+                 type: "GET",
+                 success: function(result){
+                     let error = "A user is already registered with the email address"
```

Comment:
This seems fragile, would prefer a structured error response from /registration/update when dealing with a JSON request